ios/FluentUI/Tab Bar/TabBarTokens.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `TabBar`.
+open class TabBarTokens: ControlTokens {
+    /// The height of the `TabBar` when in portrait view on a phone
+    open var phonePortraitHeight: CGFloat { 48.0 }
+
+    /// The height of the `TabBar` when in landscape view on a phone
+    open var phoneLandscapeHeight: CGFloat { 40.0 }
+
+    /// The height of the `TabBar` when on a non-phone device
+    open var padHeight: CGFloat { 48.0 }
+
+    /// Defines the background color of the  of the `TabBarItem`.
+    open var unselectedColor: DynamicColor {
+        return DynamicColor(light: ColorValue(0x6E6E6E) /* gray500 */,
+                            lightHighContrast: ColorValue(0x303030) /* gray700 */,
+                            dark: ColorValue(0x919191) /* gray400 */,
+                            darkHighContrast: ColorValue(0xC8C8C8) /* gray200 */)
+    }
+
+    /// The vertical spacing of the `TabBarItem` within the TabBar
+    open var spacingVertical: CGFloat { 3.0 }
+
+    /// The horizontal spacing of the `TabBarItem` within the TabBar
+    open var spacingHorizontal: CGFloat { 8.0 }
```

Comment:
Could we use globalTokens.spacing[.xSmall]?

Reply:
Looks like this hasn't been updated?